lnbits/templates/components/lnbits-manage-extension-list.vue

```diff
@@ -19,7 +19,7 @@
       v-for="extension in userExtensions"
       :key="extension.code"
       clickable
-      :active="extension.isActive"
+      :active="$route.path == extension.url"
```

Comment:
will this work for `my_extension/some_stuff`?
isn't `startsWith` better?

Reply:
i think startswith is better